portfolio/pom.xml

```diff
@@ -23,6 +23,12 @@
       <version>4.0.1</version>
       <scope>provided</scope>
     </dependency>
+    
+    <dependency>
+      <groupId>com.google.code.gson</groupId>
+      <artifactId>gson</artifactId>
+      <version>2.8.6</version>
+    </dependency>
```

Comment:
nit - i think it will be good practice to keep the dependencies ordered alphabetically by groupId, artifactId. that way it will be easier to find exactly what we're looking for...

Reply:
thanks! updated!